larky/src/test/resources/vendor_tests/iso8583/test_cacao_payload.star

```diff
@@ -0,0 +1,968 @@
+load("@stdlib//binascii", unhexlify="unhexlify", hexlify="hexlify")
+load("@stdlib//builtins", "builtins")
+load("@stdlib//unittest", "unittest")
+load("@vendor//asserts", "asserts")
+load("@vendor//iso8583/decoder", Decoder="decoder")
+load("@vendor//iso8583/encoder", Encoder="encoder")
+
+
+def CacaoTestCase_test_decode_encode():
```

Comment:
What is `Cacao`?

Reply:
We are currently doing ISO8583 integration for Tribal. Cacao is their payment provider, as far as I understand

Reply:
Mind if we rename the tests to something a bit more generic (ie test length encoding) instead of naming vendors or customers in the test?

Reply:
I can rename it to `Base24TestCase_test_decode_encode`, for example

according to docs from cacao, the format of ISO8583 is Base24 here (maybe with few differences)

the doc itself is in this ticket:
https://verygoodsecurity.atlassian.net/browse/CSG-116 (attachment: `1. Cacao - Manual ISO8583 v3.1.pdf`)